conf/uppmax.config

```diff
@@ -10,31 +10,34 @@ vim: syntax=groovy
  */
 
 process {
-  executor = 'local'
-  cpus = { 1 * task.attempt }
-  memory = { 8.GB * task.attempt }
-  time = { 2.h * task.attempt }
+  executor = 'slurm'
   clusterOptions = { "-A $params.project ${params.clusterOptions ?: ''}" }
   errorStrategy = { task.exitStatus == 143 ? 'retry' : 'finish' }
   maxRetries = 3
   maxErrors = '-1'
 
   $fusioncatcher{
     module = ['bioinfo-tools', 'FusionCatcher/0.99.5a']
-    cpus = { 16 * task.attempt }
-    memory = { 32.GB * task.attempt }
+    time = { 72.h * task.attempt }
+    //cpus = { 8 * task.attempt }
+    cpus = { 14 }
   }
   $star_fusion{
     module = ['bioinfo-tools', 'star-fusion']
-    cpus = { 4 * task.attempt }
-    memory = { 32.GB * task.attempt }
+    time = { 30.h * task.attempt }
+    cpus = { 8 * task.attempt }
   }
+  $fusioninspector{
+    module = ['bioinfo-tools', 'star-fusion', 'trinity/2014-07-17', 'samtools/1.5', 'htslib/1.5', 'perl_modules']
+    time = { 48.h * task.attempt }
+    cpus = { 8 * task.attempt }
+    }
 }
 
 params {
   references{
-    fusioncatcer_data = '/sw/apps/bioinfo/FusionCatcher/0.99.5a/milou/data/ensembl_v83/'
-    star_fusion_reference  = '/proj/b2013064/private/nobackup/rickard/STAR_fusion_refrence/GRCh37_gencode_v19_CTAT_lib_July272016_prebuilt'
-    fusioncatcher_data_dir = '/sw/apps/bioinfo/FusionCatcher/0.99.5a/milou/data/current'
+    params.fusioncatcer_data = '/sw/apps/bioinfo/FusionCatcher/0.99.5a/milou/data/ensembl_v83/'
```

Comment:
`params` within `params` scope - this looks wrong.

Also, `fusioncatcer` -> `fusioncatcher`